src/main/java/com/code_intelligence/jazzer/android/BUILD.bazel

```diff
@@ -1,20 +1,65 @@
 load("//bazel:compat.bzl", "SKIP_ON_WINDOWS")
+load("//bazel:jar.bzl", "strip_jar")
+
+java_import(
+    name = "jazzer_bootstrap_android_import",
+    jars = [
+        "//src/main/java/com/code_intelligence/jazzer/runtime:jazzer_bootstrap",
+    ],
+    tags = ["manual"],
+    target_compatible_with = SKIP_ON_WINDOWS,
+)
+
+android_library(
+    name = "jazzer_bootstrap_android_lib",
+    tags = ["manual"],
+    target_compatible_with = SKIP_ON_WINDOWS,
+    visibility = [
+        "//src/main/java/com/code_intelligence/jazzer/agent:__pkg__",
+    ],
+    exports = [
+        ":jazzer_bootstrap_android_import",
+    ],
+)
+
+android_binary(
+    name = "jazzer_bootstrap_android_bin",
+    manifest = "//launcher/android:android_manifest",
+    min_sdk_version = 26,
+    tags = ["manual"],
+    target_compatible_with = SKIP_ON_WINDOWS,
+    deps = [
+        ":jazzer_bootstrap_android_lib",
+    ],
+)
+
+strip_jar(
```

Comment:
Is this used for anything other than a copy? If so, I would recommend using `copy_file` instead (the repo already contains a usage you could copy).